python/src/FunctionalChaosRandomVector_doc.i.in

```diff
@@ -1,8 +1,8 @@
 %feature("docstring") OT::FunctionalChaosRandomVector
-"Functional chaos random vector used to evaluate the Sobol indices.
+"Functional chaos random vector.
 
-Available constructors:
-    FunctionalChaosRandomVector(functionalChaosResult)
+Allows to simulate a variable through a chaos decomposition,
+and retrieve its mean and covariance analytically from the chaos coefficients.
```

Comment:
```suggestion
and retrieve its mean and covariance analytically from the polynomial chaos expansion coefficients.
```

Reply:
@mbaudin47 Even if the use of a polynomial basis is the most common, don't forget that the algorithm can work with any orthogonal basis. OT provides two such bases, see `FourierSeriesFactory` and `HaarWaveletFactory`.